pkg/postprocess/validatePassword.go

```diff
@@ -17,29 +17,36 @@
 package postprocess
 
 import (
+	"github.com/americanexpress/earlybird/pkg/utils"
 	"regexp"
+	"strconv"
 	"strings"
 )
 
 var (
-	pswdPattern = regexp.MustCompile(pswdRegex)
+	pswdPattern      = regexp.MustCompile(pswdRegex)
+	splitPswdPattern = regexp.MustCompile(splitPswdRegex)
 )
 
-//PasswordFalse Make sure we don't report strings as passwords if they contain indicators that it's a variable assignment,
+// PasswordFalse Make sure we don't report strings as passwords if they contain indicators that it's a variable assignment,
 // or some other disqualifier based on line context
 func PasswordFalse(password string) (confidence int, ignore bool) {
 	var quoted int
 	confidence = 3 // Mark default confidence medium
-	//Seperate password from full value
+	// Separate password from full value
 	passwords := pswdPattern.FindStringSubmatch(password)
+	// FIXME the conditional >2 will never evaluate to true with the "current" pswdRegex.
```

Comment:
We should probably reword this  since this is actually not a FIXME but more of how our regex is.

Reply:
updated to a note